Unite.Indices/Entities/Projects/TranscriptomicStatsIndex.cs

```diff
@@ -0,0 +1,6 @@
+namespace Unite.Indices.Entities.Projects;
+
```

Comment:
We will have 2 transcriptomics data types available: bulk and single cell.
Could you create two classes from this one and name them: BulkGeneExpStats and CellGeneExpStats (short names).

BulkGeneExp will have only one property: int Total;
CellGeneExp will have more properties:
int Total - total number of donors with this data, e.g. 17
Stat<int>[] ByCellsNumber - categories by number of cells in a sample, e.g. [ {"1-100": 5}, "100-1000: 12"} ]
Stat<int>[] ByGenesModel - categories by genes model in a sample e.g. [ {"10xGenomics": 16}, {"Other": 1} ]